resources/views/Staff/poll/index.blade.php

```diff
@@ -40,8 +40,17 @@
                             <td><a href="{{ url('/dashboard/polls/' . $poll->id) }}">{{ $poll->title }}</a></td>
                             <td>{{ date('d M Y', $poll->created_at->getTimestamp()) }}</td>
                             <td>
-                                <a href="#" class="btn btn-warning">@lang('common.edit')</a>
-                                <a href="#" class="btn btn-danger">@lang('common.delete')</a>
+{{--                                <a href="#" class="btn btn-warning">@lang('common.edit')</a>--}}
```

Comment:
These commented out buttons can be removed.

Reply:
Done.